src/Sylius/Bundle/MailerBundle/Form/Type/EmailTranslationType.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\MailerBundle\Form\Type;
+
+use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Email Translation type.
+ *
+ * @author Manuel Gonzalez <mgonyan@gmail.com>
+ */
+class EmailTranslationType extends AbstractResourceType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add(
+                'subject',
+                'text',
+                array(
+                    'label' => 'sylius.form.email.subject',
+                )
+            )
```

Comment:
Or even for >PHP5.4 it can be in one row: `->add('subject', 'text', ['label' => 'sylius.form.email.subject'])`